helm/exporter-kube-state/templates/clusterrole.yaml

```diff
@@ -24,25 +26,52 @@ rules:
   - limitranges
   - persistentvolumeclaims
   - persistentvolumes
+  - namespaces
   - endpoints
-  verbs: ["list", "watch"]
-- apiGroups: ["extensions"]
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - extensions
   resources:
   - daemonsets
   - deployments
   - replicasets
-  verbs: ["list", "watch"]
-- apiGroups: ["apps"]
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - apps
   resources:
   - statefulsets
-  verbs: ["list", "watch"]
-- apiGroups: ["batch"]
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - batch
   resources:
   - cronjobs
   - jobs
-  verbs: ["list", "watch"]
-- apiGroups: ["autoscaling"]
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - autoscaling
   resources:
   - horizontalpodautoscalers
-  verbs: ["list", "watch"]
+  verbs:
+  - list
+  - watch
+- apiGroups:
```

Comment:
I'm wondering where does this values come from? I didn't see it in the kube-state-metrics repo

Reply:
it's for the kube-rbac-proxy sidecar

Reply:
Btw, the kube-rbac-proxy sidecar is missing in the helm chart.

Reply:
@pmontanari can you also add kube-rbac-proxy in this PR?

Reply:
Ok i'll do

Reply:
Well... There are a lot of differences between Helm version and  "standard" version: PodSecurityPolicies, namespaceSelector,...
I don't feel like updating all of it... sorry

Reply:
Any update on this PR? Without these resources in the cluster role, I am getting a lot of noise in the logs
example: reflector.go:205] k8s.io/kube-state-metrics/collectors/collectors.go:62: Failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:monitoring:kube-prometheus-exporter-kube-state" cannot list configmaps at the cluster scope